### PR TITLE
changes in parent controller

### PR DESCRIPTION
**Pull request by shagi9**

### CODE REVIEW

File: src/YPS.WebUI/Controllers/ParentsController.cs [hunk 1]
```diff
@@ -8,32 +8,36 @@
...
-            }
-            return BadRequest($"Unatorized");
+            //var claim = User.Claims.FirstOrDefault(x => x.Type.Equals("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname", StringComparison.InvariantCultureIgnoreCase));
+            //if (claim != null)
+            //{
+            //    var viewModel = await Mediator.Send(new GetParentsBySchoolQuery { Id = long.Parse(claim.Value) });
+            //    return Ok(viewModel);
+            //}
+            //return BadRequest($"Unatorized");1
+
```
Comment (DmMov) [inline src/YPS.WebUI/Controllers/ParentsController.cs]: Get rid of this!

**Review by vunt1k — DISMISSED** [review]

File: src/YPS.Application/Parents/Queries/GetParentProfileInfo/GetParentProfileInfoQuery.cs [hunk 2]
```diff
@@ -12,10 +12,10 @@
 
 namespace YPS.Application.Parents.Queries.GetParentProfileInfo
 {
-    public class GetParentsBySchoolQuety : IRequest<GetParentProfileInfoVm>
+    public class GetParentsProfileInfoQuery : IRequest<GetParentProfileInfoVm>
     {
         public long Id;
-        public class GetParentProfileInfoQueryHandler : IRequestHandler<GetParentsBySchoolQuety, GetParentProfileInfoVm>
+        public class GetParentProfileInfoQueryHandler : IRequestHandler<GetParentsProfileInfoQuery, GetParentProfileInfoVm>
```
Comment (DmMov) [inline src/YPS.Application/Parents/Queries/GetParentProfileInfo/GetParentProfileInfoQuery.cs]: Set the name of the class to GetParentProfileInfoQuery, you're going to get the information of only one parent

**Review by Karonuk — DISMISSED** [review]

File: src/YPS.Application/Parents/Queries/GetParentProfileInfo/GetParentProfileInfoQuery.cs (hunk 2 above)
Comment (DmMov) [reply src/YPS.Application/Parents/Queries/GetParentProfileInfo/GetParentProfileInfoQuery.cs]: You did well with the handler name but not with the query

**Review by vunt1k — APPROVED** [review]

**Review by DmMov — COMMENTED** [review]
I guess you didn't check the controller having changed the name of the query...

**Review by DmMov — APPROVED** [review]
Bless you!

**Review by Karonuk — APPROVED** [review]

**Pull request merged**